lib/primer/view_components/version.rb

```diff
@@ -5,7 +5,7 @@ module ViewComponents
     module VERSION
       MAJOR = 0
       MINOR = 0
-      PATCH = 4
+      PATCH = 5
```

Comment:
I'd like this change to be in a different PR if you don't mind
Also, could you add a `PopoverComponent` entry to the `CHANGELOG`?

Reply:
Oh! Yep, I'll remove this version bump and modify the changelog in this branch instead.

Reply:
Version change removed in https://github.com/primer/view_components/pull/48/commits/8c6b953dfeb705d42677d68f17132d6211d056f1, changelog updated in https://github.com/primer/view_components/pull/48/commits/8c57c8c972785ec87a10685bd13355d48a0f1463. I also added a line for `box_shadow` to the readme in https://github.com/primer/view_components/pull/48/commits/dc4d5f86f4d9aff3edd7ce3d7810d9ebe2696a85.